QUEST_LV_0100.tsv

```diff
@@ -2335,13 +2335,13 @@ QUEST_LV_0100_20150317_002334	Sincerity is our strength. {nl} There will be no p
 QUEST_LV_0100_20150317_002335	I think being confined in there damaged my throat and now my throat hurts. {nl} But what about the girl who was with us? Where did she go?
 QUEST_LV_0100_20150317_002336	Aren't you the Revelator who saved us last time? {nl} We were more than glad to see you back then.
 QUEST_LV_0100_20150317_002337	The Goddess must have sent you.{nl}If that's not it.. we already.. It's too scary to think about it.
-QUEST_LV_0100_20150317_002338	It was horrible in captivity. {nl} The Vubbes would come and smell us.. {nl} and make us dig here and there using a pickaxe..
+QUEST_LV_0100_20150317_002338	It was horrible in captivity. {nl} The Vubbes would come and smelled us.. {nl} and made us dig here and there using pickaxes..
```

Comment:
It was horrible in captivity. {nl} The Vubbes would come and smell us.. {nl} and make us dig here and there using pickaxes..